.travis.yml

```diff
@@ -0,0 +1,19 @@
+if: tag IS blank
+
+language: java
+
+sudo: false
+
+before_install: wget https://raw.githubusercontent.com/sormuras/bach/master/install-jdk.sh
```

Comment:
Why do we have to check for the latest EA builds instead of using standard `oraclejdk8`?